src/main/java/net/minecraftforge/network/MCRegisterPacketHandler.java

```diff
@@ -76,6 +77,16 @@ private Set<ResourceLocation> bytesToResLocation(byte[] all) {
             }
             return rl;
         }
+
+        /**
+         * Provides only the registered locations that the remote side has sent.
+         * This is useful for interacting with other modloaders via the network to inspect registered network ids.
+         *
+         * @return - The remote locations
```

Comment:
Just a bit of cleanup on the javadocs
```suggestion
         * {@return the unmodifiable set of channel locations sent by the remote side}
         * This is useful for interacting with other modloaders via the network to inspect registered network channel IDs.
```

Reply:
Sounds good!